src/components/__snapshots__/ErrorModal.spec.tsx.snap

```diff
@@ -71,7 +71,8 @@ exports[`Modal matches snapshot 1`] = `
         className="sc-jqUVSM bCBcaQ"
       >
         <button
-          className="sc-bczRLJ fLTvYy"
+          className="sc-bczRLJ kAqnNz"
+          color="orange"
```

Comment:
i think styled-components is not filtering this out because `color` is a legitimate html attribute, if you change the prop name it should be properly removed